GiraffeApp/Program.fs

```diff
@@ -35,27 +36,33 @@ let pageHandler (title: string)(view: XmlNode list)(next: HttpFunc)(ctx: HttpCon
     htmlNodes nodes next ctx
     
 [<CLIMutable>]
-type CountPayload = { Count: int }
+type IncrementForm = { Count: int }
 
-let incrementHandler: HttpHandler =
-    let culture = CultureInfo.CreateSpecificCulture("en-US")
-    bindQuery<CountPayload> (Some culture) (fun payload -> 
-        htmlNodes (Views.counter payload.Count)
-    )
+let incrementHandler(next: HttpFunc)(ctx: HttpContext): HttpFuncResult =

```

Comment:
Teerawat, please review lines 20 through 60 in this file.  incrementHandler correctly validates the antiforgery token, and embeds the token in the form, but not in a way that's convenient to re-use in another handler.

I wish to refactor the code so the logic between lines 42 through 51 (minus the call to Views.counter) is easily reusable.